onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -2280,7 +2280,7 @@ MLAS_FLOAT32X4
 MlasMultiplyAddFloat32x4(MLAS_FLOAT32X4 Vector1, MLAS_FLOAT32X4 Vector2, MLAS_FLOAT32X4 Vector3)
 {
 #if defined(MLAS_NEON_INTRINSICS)
-    return vmlaq_f32(Vector3, Vector1, Vector2);
+    return vfmaq_f32(Vector3, Vector1, Vector2);
```

Comment:
If this is needed only for newer GCC versions, can we add some GCC version check with comments describing the substitution ? Also, any idea how this change will affect Windows (MSVC) ?